index.html

```diff
@@ -160,14 +158,104 @@ <h1 class="multipost_text">Multi-post <br> stories</h1>
+                              Get my resume
+                          </button>
+                  </div>
                 
+                  <div class="skill">
+                    <div class="listtag ">
+                      <p class="listname">Languages</p>
+                  </div>
+                      <div class="listfram ">
+                         
+                          <ul class="skillul ">
+                              <li class="skillli ">
+                                  <div class="listimg"><img src="./images/FrameJava.svg" alt="javascript"></div>
+                              </li>
+                              <li class="skillli ">
+                                  <div class="listimg"><img src="./images/HTMLFrame10.svg" alt="html"></div>
+                              </li>
+                              <li class="skillli ">
+                                  <div class="listimg"><img src="./images/CSSFrame11(1).svg" alt="CSS"></div>
+                                  
+                              </li>
+                          </ul>
+                
+                          <div class="listfram ">
+                              <div class="listtag ">
+                                  <button type="button" class="icon"><img src="./images/F1.svg.svg" alt="button2"></button>
+                              </div>
+                              <ul class="skillul">
+                              </ul>
+                          </div>
+                          <div class="listfram ">
+                              <div class="listtag ">
+                          <button type="button" class="icon"><img src="./images/Frame.svg" alt="button"></button>
+                              </div>
+                              <ul class="skillul">
+                              </ul>
                           </div>
+                
                       </div>
-                    </section>                              
+                  </div>
+                </section>
+
+                <footer>
+                    <div class="Background-image">
+                        <h4 class="Contact-me">Contact me</h4>
+                        <p class="contact-details">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+                           <form action="https://formspree.io/f/moqrrblw" method="post">
+                 
+                          <input type="text" name="username" id="name" placeholder="Your name" maxlength="30" required/>
+                          <input type="email" name="email" id="email" placeholder="Your email id" required/>
+                          <textarea name="message" id="message" placeholder="Type your message" rows="8" maxlength="500"></textarea>
+                          <button type="submit" id="contact-btn">Get in touch</button>
```

Comment:
- [ ] kindly add the hover interaction on the button and add the size so it looks like the figma version.